### PR TITLE
New package: fuyoo.BSRDC version 1.1.1

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171872)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files (x86)\BSRDC\BS Redis Desktop Client.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\BSRDC\BS Redis Desktop Client.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files (x86)\BSRDC\BS Redis Desktop Client.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\BSRDC\BS Redis Desktop Client.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 895.)

File: manifests/f/fuyoo/BSRDC/1.1.1/fuyoo.BSRDC.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: fuyoo.BSRDC
+PackageVersion: 1.1.1
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2022-01-14
+Installers:
```
Comment (stephengillie) [inline manifests/f/fuyoo/BSRDC/1.1.1/fuyoo.BSRDC.installer.yaml:9]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @SpecterShell,

Apparently, `VCRedist` is a dependency here:
image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978334972
> 2024-09-10 01:49:04.722 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33810.0

Automated error analysis suggests -1978334972 may mean This package has a dependency missing from your system.

(Automated response - build 895.)

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
Verify:
image
I couldn't find how to have the application display its version number.

**Pull request merged**